test/fixtures/rule_results.yml

```diff
@@ -1,11 +1,13 @@
 # Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
 
 one:
-  host_id: one
-  rule_id: one
+  host: one
+  rule: one
+  test_result: one
```

Comment:
This is how to define relations in fixtures. Before, host and rule were just set to `nil`. This required updates to only 2 other tests which were relying on host and rule to be `nil`.